### DISCUSSION TITLE
Why my stars decrease from 100 to 35 in only one night？

### DISCUSSION THREAD

**Discussion by SkyWorkAIGC** [community/community#46815] [New to GitHub]
### Select Topic Area

Question

### Body

The number of stars of my repository "SkyText-Chinese-GPT3" was 100 on 3rd Feb. 
But on 6th Feb, it decreased to 35.
![image](https://user-images.githubusercontent.com/120169448/217723173-e3f0de91-65d0-4d9e-b864-855659bd02d5.png)
So I wonder what's wrong with my repository? 
Ask you for your opinions.

**Comment by jge162** [comment]
Sorry I don’t think we can help you here. Gotta. Ask GitHub support.

**Reply by SkyWorkAIGC** [reply]
> Sorry I don’t think we can help you here. Gotta. Ask GitHub support.

Could you please teach me how to connect with "GitHub support"? It's hard to find them.

**Reply by jge162** [reply]
@SkyWorkAIGC here is the [link](https://support.github.com/request/account?tags=hubberfy_account). Just pick other issue.

**Comment by git-sumana** [comment]
Response #1
You might have deleted a repository that have those stars

**Comment by git-sumana** [comment]
Response #2
You might have changed the visibility from public to private

**Comment by git-sumana** [comment]
Mark as answered, so that it would be helpful for others too! Thanks in advance
